src/Discord.Net.WebSocket/DiscordSocketConfig.cs

```diff
@@ -124,6 +125,16 @@ public class DiscordSocketConfig : DiscordRestConfig
         /// </summary>
         public bool GuildSubscriptions { get; set; } = true;
 
+        /// <summary>
+        ///     Gets or sets the gateway limits.
+        ///     <note type="warning">
```

Comment:
Move this to `<remarks>`. Replace `It` with `This property`.

Reply:
I believe this warning should stay in the summary because VS doesn't show remarks (or it's disabled by default) and wanted to make who is using the lib aware that they shouldn't change the limits unless they know what they are doing, it might have really bad consequences.

Reply:
VS 16.6 released just the other day which shows the Remarks section in Quick Info's. I didn't need to change any settings for that to come up.

Reply:
Updated from 16.5 and it seems they added remarks, returns, parameters, etc to 16.6. Going to move it to remarks.